understanding/20/error-identification.html

```diff
@@ -72,21 +72,18 @@ <h2>Intent of Error Identification</h2>
          that user agents or assistive technologies can use to identify an error and provide
          error information to the user. For example, certain technologies can specify that
          the user's input must not fall outside a specific range, or that a form field is required.
-         Currently, few technologies support this kind of programmatic information, but the
-         Success Criterion does not require, nor prevent it.
-         
+         This type of programmatic information is not required for this success criterion, but may be covered
+         by other criteria such as <a href="#name-role-value">4.1.2 Name, Role, Value</a>.
```

Comment:
The idea of 3.3.1 is: **ALL** users should know that some error has occured. We have always thought it appropriate to cover missing programmatic linking of error messages, and programmatic identification of required state where indiated visually, under 3.3.1 since it is an aspect of the use case of realising that there has been an error. Is the added reference to 4.1.2 meant to indicate that that kind of error should be reported in 4.1.2 and not here? If so, that is not explicit now. I guess programmatic identification of errors might possibly be done in several ways (programmatically moving focus to the message?) that would not **necessarily** fall under 4.1.2. Another way would be 4.1.3, which is not referenced here - I guess it should be, by analogy.

I can see the advantage of covering aspects of bad errror handling under other SCs: 4.1.2 (messages not programmatically referenced) 1.3.1 ('required' indicated only visually), 1.3.2 (error message appears above the field), 4.1.3 (error status message not available) - BUT from the point of use of an audit report, this means the issues around error handling are widely dispersed across the report. I'm OK with that (because what it changes is nothing substantial, but only the allocation of errors to SCs), but I wonder if at least the programmatic availability of error messages is something that can usefully be reported under 3.3.1 (because it does not concern so much **the value / state of a UIC** but the availability / identification of the result of a user interaction).

Reply:
> Finally it is fairly clearly established that we do not create a catalog of ways something could fail other SCs in the Understanding documents. I think the balance created here is reasonable.

Yes, I see the point in that. But my point was that there are reasons to fail unreferenced error messages **here**, in 3.3.1, as this cannot easily be captured under 4.1.2 (it is not narrowly a name, role or value of an UIC that is at fault) and arguably 1.3.1 is met if output is available in browse mode. And it is an accessibility issue that should be captured somewhere - why not here? 

I am not in the way of progressing this PR without such addition, though, and to create a separate PR for it.

Reply:
Thank you, please open a new issue/PR to address your concerns.